dist/profile/templates/confluence/vhost.conf

```diff
@@ -2853,7 +2853,8 @@ RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector$" "https://plugins.jenkins.io
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Zulip\+Plugin$" "https://plugins.jenkins.io/zulip" [NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Remove\+Git\+Plugin\+BuildsByBranch\+BuildData$" "https://plugins.jenkins.io/git/#remove-git-plugin-buildsbybranch-builddata-script" [NE,NC,L,QSA,R=301]
-RewriteRule "^/display/JENKINS/violation\+comments\+to\+gitlab$" "https://plugins.jenkins.io/violation-comments-to-gitlab" [NC,L,QSA,R=301] RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+ RewriteRule "^/display/JENKINS/violation\+comments\+to\+gitlab$" "https://plugins.jenkins.io/violation-comments-to-gitlab" [NC,L,QSA,R=301]
```

Comment:
ack, suggestion ended up with too many spaces

can you remove them?
otherwise LGTM

Reply:
Done! I've just pushed a new commit